config/ng-config.json

```diff
@@ -82,7 +82,8 @@
       "customLocaleURL": "/bahmni_config/openmrs/i18n/",
       "loginPageUrl": "../../bahmni/home/index.html#/login",
       "patientsURL": "/bahmni/clinical/index.html#/default/patient/",
-      "patientsURLGeneralInformationTab": "/dashboard?currentTab=DASHBOARD_TAB_GENERAL_KEY"
+      "patientsURLGeneralInformationTab": "/dashboard?currentTab=DASHBOARD_TAB_GENERAL_KEY",
+      "appointmentsListPatientLink": "/openmrs/coreapps/clinicianfacing/patient.page?patientId={{patientId}}"
```

Comment:
@icrc-psousa that's the link that we want in our impl, I guess the Bahmni folks will want to default to something else :-)

@buvaneswari-arun can you suggest a default appointments list patient link to use here?